pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -40,6 +43,42 @@ const (
 	kubeProxyPath = k8sDir + "kube-proxy.exe"
 	// hybridOverlayPath is the location of the hybrid-overlay-node exe
 	hybridOverlayPath = k8sDir + "hybrid-overlay-node.exe"
+)
+
+// filesToTransfer is a map of what files should be copied to the Windows VM and where they should be copied to
+var filesToTransfer map[*payload.FileInfo]string
+
+// getFilesToTransfer returns the properly populated filesToTransfer map
+func getFilesToTransfer() (map[*payload.FileInfo]string, error) {
+	if filesToTransfer != nil {
+		return filesToTransfer, nil
+	}
+	srcDestPairs := map[string]string{
+		payload.IgnoreWgetPowerShellPath: remoteDir,
+		payload.WmcbPath:                 k8sDir,
+		payload.HybridOverlayPath:        k8sDir,
+		payload.HNSPSModule:              remoteDir,
+		payload.WindowsExporterPath:      k8sDir,
+		payload.FlannelCNIPluginPath:     cniDir,
+		payload.WinBridgeCNIPlugin:       cniDir,
+		payload.HostLocalCNIPlugin:       cniDir,
+		payload.WinOverlayCNIPlugin:      cniDir,
+		payload.KubeProxyPath:            k8sDir,
+		payload.KubeletPath:              k8sDir,
+	}
+	files := make(map[*payload.FileInfo]string)
+	for src, dest := range srcDestPairs {
+		f, err := payload.NewFileInfo(src)
+		if err != nil {
+			return nil, errors.Wrapf(err, "could not create FileInfo object for file %s", src)
+		}
+		files[f] = dest
+	}
+	filesToTransfer = files
+	return filesToTransfer, nil
+}
+
+const (
```

Comment:
Why are you creating two sets of `const`? Please keep them as one and move the code you are introducing after `var log = logf.Log.WithName("windows")`